apps/flipper/ios/ci_scripts/ci_post_clone.sh

```
@@ -4,7 +4,7 @@
 set -e
 
 # Adjust the base path to the correct root folder
-BASE_PATH="$(cd "$(dirname "$SRCROOT")/../../../../" && pwd)"
+BASE_PATH="$(cd "$(dirname "$SRCROOT")"/../../../../" && pwd)"
```

Shellcheck (0.10.0)

[error] Couldn't parse this command expansion. Fix to allow more checks.

(SC1073)